src/System.Management.Automation/engine/Modules/TestModuleManifestCommand.cs

```diff
@@ -131,6 +131,22 @@ protected override void ProcessRecord()
                             }
                         }
 
+                        //RootModule can be null, empty string or point to a valid .psm1, or .dll.  Anything else is invalid.
```

Comment:
I agree. We don't even have the desired method. Perhaps even better, make a request in .Net.